tests/testthat/test-printing.R

```diff
@@ -333,7 +333,7 @@ test_that("Inset works for table, ref_footnotes, and main footer", {
 
   expect_false(result[[1]]) # No inset
   expect_true(result[[2]]) # Inset
-  expect_true(all(vec_tt[sep_index + 1] == "   ===================="))
+  expect_true(all(vec_tt[sep_index + 1] == "   ======================"))
```

Comment:
Why has this changed?

Reply:
It shouldn't have as far as I know, if the body of the test hasn't changed...

Reply:
Ah, the body of the test did change. You're (correctly) calling matrix_form(tt, TRUE) now. this makes the table wider, so the divider is wider.